packages/near-membrane-dom/src/index.ts

```diff
@@ -1 +1,2 @@
 export { default } from './browser-realm';
+export { VirtualEnvironment } from '@locker/near-membrane-base';
```

Comment:
this is the most important pieces (API change) in this PR, it allows other packages to create arbistrary virtual environment. The primary case here being the creation of a connection between an iframe.contentWindow and an existing red sandbox, what we call the magenta environment.